tests/e2e/specs/in-game/layout.spec.js

```diff
@@ -17,6 +17,15 @@ describe('Game View Layout', () => {
     cy.get('[data-cy=nav-drawer]').should('not.be.visible');
   });
 
+  it('Should display usernames of player and opponent', () => {
+    cy.get('[data-cy=opponent-username]').should('be.visible');
+    cy.get('[data-cy=player-username]').should('be.visible');
+  });
+
+  it('Should truncate usernames > 12 characters', () => {
+    cy.get('[data-cy=opponent-username]').should('have.text', 'definitelyNo...');
+  });
```

Comment:
If we refactor the truncation to work with css I don't think this test will be practical because the number of characters will be so variable based on viewport width. I like the idea, but we may need to let this one go in order to maximize the responsive flexibility we'll get out of letting CSS do it's magic